doc/design/overview.md

```diff
@@ -1,6 +1,6 @@
 # Loaders Design
 
-There are currently the following [loader hooks](https://github.com/nodejs/node/tree/master/doc/api/esm.html#esm_hooks):
+There are currently the following [loader hooks](https://github.com/nodejs/node/tree/master/doc/api/esm.html#hooks):
```

Comment:
```suggestion
There are currently the following [loader hooks](https://github.com/nodejs/node/blob/master/doc/api/esm.md#hooks):
```

Close, but the pre-rendered documents are still Markdown as they have not yet been compiled.